test/e2e/multihoming.go

```diff
@@ -1812,3 +1930,75 @@ func createMultiNetworkPolicy(mnpClient mnpclient.K8sCniCncfIoV1beta1Interface,
 	)
 	return err
 }
+
+func computeIPWithOffset(baseAddr string, increment int) (string, error) {
```

Comment:
We don't need this function since there is a `generator/ip/ip_generator.go` package for this

Reply:
ops I had no idea

Reply:
In the end I imported the package you suggested, but in general we should avoid as much as possible to import code from ovn-k, since that's what we're testing here :)